proto/ibc/transfer/transfer.proto

```diff
@@ -60,3 +60,13 @@ message DenomTrace {
   // base denomination of the relayed fungible token.
   string base_denom = 2;
 }
+
+// Params defines the set of IBC transfer parameters.
+// NOTE: To prevent a single token from being transferred, set the TransfersEnabled parameter to
```

Comment:
I think this note could be slightly more clear:
* to block any cross-chain transfers -> disable ibc-transfer param
* to block intrachain transfers for a specific denom -> disable bank param for that denom